tests/packages.sh

```diff
@@ -17,11 +17,25 @@ docker image inspect "$research_template_image" > /dev/null
 # some packages are installed multiple times with different version numbers.
 # See https://github.com/opensafely-core/research-template-docker/issues/22
 
-# These packages are known to be extra to the local installation.
+# These packages are known to be extra to the local installation. Must be sorted
```

Comment:
does "extra to the local installation" still make sense with the base packages in here now?

Reply:
Fair point, but given the implementation of this uses bash variables with sort/comm, I'd be leaving a *different* problem for future travellers!

I'll give it a go, see if I can make it readable

Reply:
Ok, I figured out how to generate packages in an renv.lock from the devcontainer, so we're comparing apples to apples.